script/integration-tests

```diff
@@ -0,0 +1,33 @@
+#!/bin/bash
+# Special case: if ./ksonnet-lib exists, set KUBECFG_JPATH
+test -d $PWD/ksonnet-lib && export KUBECFG_JPATH=$PWD/ksonnet-lib
+
+# We require below env
+: ${GOPATH:?} ${KUBECFG_JPATH:?}
+
+# Default to "dind" for testing
+INTEGRATION_TESTS_CTX=${1:-dind}
+
+set -xu
+
+k8s_create_dind() {
```

Comment:
From a few lines below, just after the function ends:
```
kubectl get nodes --context=${INTEGRATION_TESTS_CTX} || k8s_create_${INTEGRATION_TESTS_CTX}
```
I moved as func to leave script's _main_ focusing on the actual logic
(vs bringing up the dind cluster), unsure if worth moving to `libtest.sh`.